crates/ty_python_semantic/src/types/diagnostic.rs

```diff
@@ -3140,6 +3141,56 @@ pub(super) fn hint_if_stdlib_submodule_exists_on_other_versions(
     add_inferred_python_version_hint_to_diagnostic(db, &mut diagnostic, "resolving modules");
 }
 
+/// This function receives an unresolved `foo.bar` attribute access,
+/// where `foo` can be resolved to have a type but that type does not
+/// have a `bar` attribute.
+///
+/// If the type of `foo` has a definition that originates in the
+/// stsnfstf library and `foo.bar` *does* exist as an attribute on *other*
```

Comment:
```suggestion
/// stsnfstf library and `foo.bar` *does* exist as an attribute on *other*
```

The what library? `stsnfstf`